server/apps/main/templates/main/confirmation_page.html

```diff
@@ -0,0 +1,322 @@
+
+    div.confirm_success a.h5 {
+      float: right;
+    }
+
+    div.what_happens{
+      float: right;
+    }
+
+    div.what_happens a{
+      color: #000;
+      text-decoration: none;
+      font-size: 18px;
+    }
+    
+    span.step{
+      font-size: 14px !important;
+    }
+
+    div.what_now {
+      text-align: center;
+      margin-top: 40px;
+    }
+
+    div.what_now p.h4 {
+      font-size: 1.4rem;
+      padding-bottom: 10px;
+      cursor: pointer;
+    }
+
+    div.what_now ul.texts {
+      padding-left: 0%;
+      padding: 20px;
+      background-color: #E8F3FC;
+    }
+
+    div.what_now ul.texts li.text {
+      list-style: none;
+    }
+
+    span.asterik {
+      font-weight: 600;
+    }
+
+    div.to_do_now {
+      text-align: center;
+      margin-top: 60px;
+    }
+
+    div.to_do_now p.h4 {
+      font-size: 1.5rem;
+      font-weight: 550;
+    }
+
+    div.btn_flex_one,
+    div.btn_flex_two {
+      display: flex;
+      max-width: 500px;
+      margin: auto;
+      justify-content: space-between;
+    }
+
+    div.btn_flex_one button,
+    div.btn_flex_two button {
+      
+    }
+
+    div.btn_flex_one button a,
+    div.btn_flex_two button a {
+      color: #000;
+      text-decoration: none;
+    }
+
+    @media screen and (max-width: 750px){
+      div.btn_flex_one,
+      div.btn_flex_two {
+        flex-direction: column;
+      }
+
+      div.btn_flex_one button,
+      div.btn_flex_two button{
+        margin-bottom: 10px;
+      }
+    }
+
+  </style>
+  <!-- Custom Styles Ends Here -->
+
+
+  <!-- Navbar Starts Here -->
+  <nav class="navbar navbar-expand">
+    <ul class="navbar-nav mx-auto justify-content-around">
+      <li class="nav-item">
+        <a class="nav-link text-dark" href="{% url "main:index" %}">About</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link text-dark" href="{% url "main:upload" %}">Share</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link text-dark" href="#">View</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link text-dark" href="#">MyPage</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link text-dark" href="{% url "main:logout" %}">Logout</a>
+      </li>
+    </ul>
+  </nav>
+  <!-- Navbar Ends Here -->
+
+
+  <!-- Container Starts Here -->
+  <div class="container">
+
+    <!-- Confirmation Section Starts Here -->
+    <section class="confirmation">
+      <div class="confirmation_flex">
+
+        <!-- Confirm SVG -->
+        <div class="confirm_svg">
+          <i class="fa fa-check" aria-hidden="true"></i>
+        </div>
+
+        <!-- Confirm Success -->
+        <div class="confirm_success">
+          <p class="h3 text-success">
+            Success!
+          </p>
+
+          <p class="h5">
+            Thank you for sharing your experience.
+          </p>
+
+          <!-- What Happens Next Dropdown Goes Here -->
+          <div class="dropdown show what_happens">
```

Comment:
Out of curiosity, does this work for you? 
Wasn't sure if you intended this to be functional, or if you wanted to add this as a template?

Reply:
> Out of curiosity, does this work for you?
> Wasn't sure if you intended this to be functional, or if you wanted to add this as a template?

Not really, I'll take them out.